ui/framework/src/test/frontstage/StandardFrontstage.test.tsx

```diff
@@ -1,243 +1,243 @@
-/*---------------------------------------------------------------------------------------------

```

Comment:
Why are every line marked as changed for this file and several other ui/framework test files?

Reply:
I wish I knew. It happens in in my merges from master as well, but when I look into the individual changes it does not show. It looks like a line ending issue but I changed that setting already.

Reply:
They are all whitespace-only changes. Probably line endings? FYI you can tell GitHub to omit whitespace changes in diffs, or use [this extension](https://chrome.google.com/webstore/detail/github-whitespace/fnpkdafamnbjoldglihkjjdicofghccm) to make it always do so.
@SpencerWBarnes something wrongly configured in your editor? Please revert such changes.
![image](https://user-images.githubusercontent.com/22944042/135867313-6b2473ba-f038-49c1-920c-62c35c7b861b.png)